mne/channels/tests/test_channels.py

```diff
@@ -449,6 +449,14 @@ def test_drop_channels():
     pytest.raises(ValueError, raw.drop_channels, ["MEG 0111", 5])
     pytest.raises(ValueError, raw.drop_channels, 5)  # must be list or str
 
+    # by default, drop channels raises a ValueError if a channel can't be found
+    m_chs = ["MEG 0111", "MEG blahblah"]
+    pytest.raises(ValueError, raw.drop_channels, m_chs)
+    # ...but this can be turned to a warning
+    pytest.warns(RuntimeWarning, raw.drop_channels, m_chs, on_missing='warn')
```

Comment:
sorry I didn't catch this before, but ideally these should use the newer context-manager style of `raises` or `warns`.  `git grep "with pytest.raises"` will show you some examples of the syntax; the advantage is (1) easier to read what's being tested and (2) checks that the actual error/warning message matches rather than just the error/warning type.

Reply:
Thanks - I've updated in [761d719](https://github.com/mne-tools/mne-python/pull/11077/commits/761d719d03cd31cd5d8e602e95f7a79149a4e7e5)